docs/_guides/codewind-eclipse-quick-guide.md

```diff
@@ -10,13 +10,13 @@ tags: Codewind, Eclipse, microservice
 ---
 
 ## Objectives
-* Install Eclipse and Codewind
-* Develop a simple microservice that uses Eclipse Codewind in Eclipse
+* Install Eclipse and Codewind.
+* Develop a simple microservice that uses Eclipse Codewind in Eclipse.
 
 ## Overview
-Use Eclipse Codewind to create application projects from `Application Stacks` that your company builds. With Codewind, you can focus on your code and not on infrastructure and Kubernetes. Application deployments to Kubernetes occur through pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.
+Use Eclipse Codewind to create application projects from Application Stacks that your company builds. With Codewind, you can focus on your code and not on infrastructure and Kubernetes. Application deployments to Kubernetes occur through pipelines when developers commit their local code to the correct Git repos Kabanero is managing via webhooks.
 
-Use Codewind to create projects based on different template types. These projects include IBM Cloud starters, OpenShift Do (odo), and Appsody templates. Today, there are templates for: IBM Cloud Starters, odo, Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, and Node.js with Loopback.
+Use Codewind to create projects based on different template types. These projects include IBM Cloud starters, OpenShift Do (odo), and Appsody templates. Today, there are templates for IBM Cloud Starters, odo, Eclipse MicroProfile/Java EE, Springboot, Node.js, Node.js with Express, and Node.js with Loopback.
```

Comment:
The first instance of "IBM Cloud starters" has a lower case "s," but the second instance has an upper case "S." We should keep capitalization consistent.